org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -27,7 +27,7 @@
             <unit id="org.eclipse.jdt.source.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
             <unit id="org.mockito.mockito-core" version="0.0.0"/>
-            <repository location="https://download.eclipse.org/eclipse/updates/4.25/"/>
+            <repository location="https://download.eclipse.org/eclipse/updates/4.26-I-builds/I20221013-1800/"/>
```

Comment:
If I understand it correctly, 4.26-I-Build should contain Java 19 support, do we need to remove the separate Java 19 patch from target platform?

https://github.com/eclipse/eclipse.jdt.ls/blob/bee7e000772e2187fee8d8aa486d32ed51d7fa65/org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target#L41-L45

Reply:
@testforstephen @jdneo You can remove the Java 19 patch. You should also remove the org.eclipse.jdt.java19patch artifact from org.eclipse.jdt.ls.tests/pom.xml. See https://github.com/eclipse/eclipse.jdt.ls/commit/ec927faca029a306c6ff93db0835d95f0d40520b and https://github.com/eclipse/eclipse.jdt.ls/blob/master/org.eclipse.jdt.ls.tests/pom.xml#L88

Reply:
Thank you @snjeza !